pint_pandas/pint_array.py

```diff
@@ -859,10 +890,17 @@ def _create_comparison_method(cls, op):
         return cls._create_method(op, coerce_to_dtype=False)
 
     @classmethod
-    def from_1darray_quantity(cls, quantity):
+    def from_1darray_quantity(cls, quantity, subdtype=None):
```

Comment:
The addition of subdtype here is noted, but none of the callers of `from_`darray_quantity` make use of it.  When I have two PintArrays (of 'EUR') for example, and I construct them as `pint[EUR][float64]` and then divide them to get a dimensionless ratio, the `[float64]` nature goes away and I get back a `pint[dimensionless][Float64]`, which I don't want.  The question is: should we fix all the callers of this function to pass the subdtype of the quantity as the subdtype value, and should that override the call on line 900 to pass the incoming `subdtype` of the `pd.array` call?  Or should we fix this function to preserve the subdtype of the `quantity.magnitude` if the incoming `subdtype` is None and pass that on line 900 rather than rederiving it on line 901?